Everything Xprint-related has been removed from libXaw-1.0.5 and
from the apps in git master.  But this macro must remain until 
new versions of all the apps are released.

---

 Makefile.am |    4 ++++
 xaw.m4      |   13 +++++++++++++
 2 files changed, 17 insertions(+)

--- origsrc/libXaw-1.0.5/Makefile.am	2008-11-07 11:15:56.000000000 -0600
+++ src/libXaw-1.0.5/Makefile.am	2008-11-20 18:59:32.628035700 -0600
@@ -1,5 +1,8 @@
 SUBDIRS = include src man
 
+aclocaldir = $(datadir)/aclocal
+aclocal_DATA = xaw.m4
+
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = 
 
@@ -20,6 +23,7 @@
 EXTRA_DIST = xaw6.pc.in \
 	xaw7.pc.in \
 	autogen.sh \
+	xaw.m4 \
 	old-doc/Changelog \
 	old-doc/CHANGES \
 	ChangeLog
--- origsrc/libXaw-1.0.5/xaw.m4	1969-12-31 18:00:00.000000000 -0600
+++ src/libXaw-1.0.5/xaw.m4	2008-11-20 18:58:03.299910700 -0600
@@ -0,0 +1,13 @@
+AC_DEFUN([XAW_CHECK_XPRINT_SUPPORT],[
+	PKG_CHECK_MODULES(TMP_XAW, xaw7, success=yes, success=no)
+
+	if test "x$success" = "xyes"; then
+		$1_CFLAGS=$TMP_XAW_CFLAGS
+		$1_LIBS=$TMP_XAW_LIBS
+
+		xaw_use_xprint=no
+		AM_CONDITIONAL([XAW_USE_XPRINT], [false])
+	else
+		AC_MSG_ERROR([Xaw7 not found])
+	fi
+])
